pyproject.toml

```diff
@@ -38,7 +38,7 @@ test = [
     'pytest>=5.1.1',
     'tox',
     'towncrier',
-    'uvicorn'
+    # 'uvicorn'
```

Comment:
What's the reason for comment this line?

Reply:
I commented `uvicorn` because I can not install it on my machine, when it is in the compilation step, `gcc` reports an error that I could not yet identify the cause